### DISCUSSION TITLE
Welcome to Hostel-Management Discussions!

### DISCUSSION THREAD

**Discussion by starkblaze01** [starkblaze01/Hostel-Management#141] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Anshul1811** [comment]
Hi
I am having difficuility in setting the SECRETKEY in keys file of my online mongodb server . Can you please guide me how to do that?
